docs/flashbots-protect/rpc/mev-share.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: MEV-Share
+---
+
+import ProtectButton from "protect-button"
+
+Flashbots Protect users' transactions automatically flow to the Flashbots [Matchmaker](/flashbots-mev-share/overview#what-is-a-matchmaker), which returns them up to 80% of the MEV that their transactions create. By default, users connected to Protect will be connected with the [Stable](/flashbots-protect/rpc/mev-share#stable-configuration) configuration, which is continuously tuned by Flashbots to optimize execution while protecting users from harmful MEV.
+
+MEV-Share users on Flashbots Protect can directly control which parts of their transaction they wish to share with searchers. Sharing less data will improve privacy, while sharing more will improve chances of both having their transactions included quickly and receiving MEV kickbacks.
```

Comment:
I may not mention quick inclusion here, but rather focus on that when we discuss orderflow sharing with other builders? I suppose it is true bc searcher bids would increase chance of inclusion but might be more straightforward not to mingle